clang/lib/CIR/CodeGen/CIRGenBuiltinAArch64.cpp

```diff
@@ -41,6 +53,16 @@ CIRGenFunction::emitAArch64SVEBuiltinExpr(unsigned builtinID,
     return mlir::Value{};
   }
 
+  mlir::Location loc = getLoc(expr->getExprLoc());
+  // Generate vscale * scalingFactor
+  auto vscaleTimesFactor = [&](int32_t scalingFactor) {
+    StringRef intrinsicName = "vscale.i64";
+    auto vscale = emitIntrinsicCallOp(builder, loc, intrinsicName,
```

Comment:
```suggestion
    StringRef intrinsicName = "vscale.i64";
    auto vscale = emitIntrinsicCallOp(builder, loc, "vscale",
```
The suffix will get added automatically during lowering to LLVM IR.

Do you think it's useful to add a `cir.vscale` operation to make this easier to identify in transformation passes?

Reply:
> The suffix will get added automatically during lowering to LLVM IR.

Addressed in this [commit](https://github.com/llvm/llvm-project/pull/172346/commits/220de86345523b63ea06525cba85bfa78774b1d2)